docs/G-Codes.md

```diff
@@ -688,6 +692,50 @@ is specified then the toolhead move will be performed with the given
 speed (in mm/s); otherwise the toolhead move will use the restored
 g-code speed.
 
+### [generic_cartesian]
+The commands in this section become automatically available when
+`kinematics: generic_cartesian` is specified as the printer kinematics.
+
+#### SET_STEPPER_KINEMATICS
+`SET_STEPPER_KINEMATICS STEPPER=<stepper_name> KINEMATICS=<kinematics>
+[DISABLE_CHECKS=[0|1]]`: Set or update the stepper kinematics.
+`<stepper_name>` must reference an existing stepper defined in
+`printer.cfg`, and `<kinematics>` describes the carriages the stepper
+moves. All defined carriages can be specified here, as well as their
+linear combinations, for example `x`, `x+y`, `y-0.5z`, `x-z`, etc. Note
+that unlike the printer configuration, `KINEMATICS=...` string must not
+use `*` symbol for multiplication, since it is a special GCode character,
+and if required, the multipliers must precede the carriage reference
+without any further delimiters, e.g. `0.5x+0.5z` or
+`0.333333333333x-0.666666666667y`.
```

Comment:
The '*' character should no longer be special (commit d45b9c92).  I think it would be preferable to require the same syntax in the g-code that is in the config.

Reply:
I updated and simplified the code to always expect '*' as a multiplier